src/Heading.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <div>
+      <header className="App-header">
+        {" "}
```

Comment:
There is no need for these: `{" "}`
It is just adding an empty string. The same goes for the ones on lines 12 and 13